.gitignore

```diff
@@ -22,6 +22,7 @@ jspm_packages/
 # gatsby files
 .cache/
 public
+*.css.d.ts
```

Comment:
Workaround but not a good solution for the current problem of every .css.d.ts file generated in the same folder of a css file